scripts/provision/configure_environment.sh

```diff
@@ -29,6 +29,15 @@ a2ensite magento2.conf
 # Disable default virtual host
 sudo a2dissite 000-default
 
+# Copy varnish vcl file
```

Comment:
This should be moved to https://github.com/paliarush/magento2-vagrant-for-developers/blob/2.0/scripts/provision/configure_environment_recurring.sh and probably adopted. We need to make sure that all existing installations will work correctly after pulling latest changes.